src/DataTransformer/ValueObjectTransformer.php

```diff
@@ -54,6 +54,9 @@ public function transform($value)
                 if ($child instanceof ButtonBuilder) {
                     continue;
                 }
+                if (!$child->getOption('mapped', true)) {
```

Comment:
`!$child->getOption('mapped')` would be enough, right?

Reply:
I can add simple tests for this case